engine-types/src/types/gas.rs

```diff
@@ -99,6 +99,14 @@ impl Mul<usize> for EthGas {
     }
 }
 
+impl Mul<EthGas> for usize {
+    type Output = EthGas;
+
+    fn mul(self, rhs: EthGas) -> Self::Output {
+        EthGas(self as u64 * rhs.0)
```

Comment:
Note conflict with PR removing `as` conversions.

Reply:
I'm aware, I just am ignoring it for now and will update either or when the time comes.